README.md

```diff
@@ -30,7 +30,6 @@ Things you may want to cover:
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
   - [🔭 Video Presentation](#video)
 - [💻 Getting Started](#getting-started)
```

Comment:
- [ ] It would be nice if you fix the Video Presentation link because right now it is broken 😨 Kindly fix it or remove it `(If you don't have any dedicated section in the readme file for this Video Presentation link)` to make your readme descriptive and professional 👍